person.rb

```diff
@@ -0,0 +1,53 @@
+class Person
+  attr_reader :age, :salary, :phase, :target_raise
+  attr_accessor :name
+
+  def initialize(options={})
+    @phase = 3
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+    # @target_raise = 1000
+    # @salary = 0
+  end
+
+  def offer_high_five
+    "High five!"
+  end
+
+  # def set_phase(num)
```

Comment:
delete methods that have been refactored. they take up space in your mind and on the computer